documentation/typespec-end-to-end-scenarios.md

```diff
@@ -68,57 +68,65 @@ classDef grey fill:#CCCCCC,color:#555555;
 User((::))-->A
 A["clone Rest-API and SDK repo locally"] --> B
 B["2.1 pre-requsite dependencies installation
-(each language would have a installation script)"] --> C
-C["2.2 TypeSpec-Project-Sync.ps1
-(a.create tsp-location.yaml and drop to temp location
-b.update tsp-location.yaml if existed
-c.fetch specs from remote spec repo or use local spec repo)"] --> D
- D["2.3 TypeSpec-Project-Generate.ps1
-(a.create scaffolding for new project
-b.update tsp-location.yaml
-c.copy typespec specs to temp folder
-d.generate sdk code)"]
+(each language would have a installation script)"] --> F
+F["2.2 TypeSpec-Project-Prcoess.ps1"] --> C
+C["2.3 TypeSpec-Project-Sync.ps1"] --> D
+D["2.4 TypeSpec-Project-Generate.ps1"]
 D-->E["code build"]
 class A,E grey
-class C,D highlight
+class C,D,F highlight
 ```
-- 2.2 `TypeSpec-Project-Sync.ps1`
+- 2.2 `TypeSpec-Project-Process.ps1`
    - input: 
-     - sdkProjectDirectory
-     - typespecProjectDirectory
-     - repo
-     - commit
-     - additionalDirectories
-     - localMode (use local spec and don't fetch from remote)
-     - localSpecRepoPath
-    Note: we might pull out tsp-location.yaml create/update part as single script to be used by #2 scenario
-   - output:
-     - path of sdkProjectDirectory
+     - typespecProjectDirectory (required)
+       either a folder of `tspconfig.yaml` or a remoteUrl of `tspconfig.yaml`
+     - commitSha (optional)
+     - repoUrl (optional)
+   - output: n/a
 
-- Function `Get-{Language}-Tsp-Location-Path`
-   - input: sdkProjectDirectory
-   - output: path of tsp-location.yaml
+- 2.3 `TypeSpec-Project-Sync.ps1`
```

Comment:
I was expecting to see the summary of what the script does in this section with the inputs and outputs.